config/yugabyte/regression_pipelines/orderby_workloads/yb_colocated/ORDG8_orderby_backwardscan_post_updates.yaml

```diff
@@ -0,0 +1,256 @@
+#100000	col1	col2	    col3      col4
+#Table1	1*1M	1000*1k	    10000*100
+
+#TEST FOR YUGABYTE REGULAR TABLES
+#GOAL7 Orderby on hash partitioned table
+#Orderby clause on non indexed column of bigint datatype
+#Orderby clause on hash pkey of bigint datatype
+#Orderby clause on hash secondary indexed column of bigint datatype
+#Orderby clause on range secondary indexed column of bigint datatype
+type: YUGABYTE
+driver: com.yugabyte.Driver
+url: jdbc:yugabytedb://{{endpoint}}:5433/yugabyte?sslmode=require&ApplicationName=featurebench&reWriteBatchedInserts=true&load-balance=false
+createdb: drop database if exists yb_colocated; create database yb_colocated with colocated=true
+username: {{username}}
+password: {{password}}
+batchsize: 128
+isolation: TRANSACTION_REPEATABLE_READ
+loaderthreads: 1
```

Comment:
loaderThreads?

Reply:
Fixed. Though I would prefer that framework to by default load data in parallel when multiple independent tables are part of YAML unless there is a dependency between tables, like Foreign keys, which is when we load tables serially in a sequence using single thread.